core/liquidity-manager/helpers/liquidity-helper.ts

```diff
@@ -0,0 +1,92 @@
+import { MasternodeInfo, MasternodeState as MnState } from '@defichain/jellyfish-api-core/dist/category/masternode';
+import { Masternode, MasternodeState } from '../../shared/dto/masternode';
+import { Withdrawal } from '../../shared/dto/withdrawal';
+import { Logger } from '../../shared/logger';
+import { Util } from '../../shared/util';
+import { Node } from '../../shared/node';
+import { Api } from '../../shared/api';
+
+export class LiquidityHelper {
+  private readonly masternodeCollateral = 20000;
+  private readonly masternodeFee = 10;
+
+  private readonly minLiquidity = 50000; // TODO
```

Comment:
maybe we should move them also into env. variables?

Reply:
Min. liquidity makes sense. The other two not really, as they are really constant. Will move it to a config.